extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -109,30 +109,40 @@ require([
 		}, onPlayerReady);
 	}
 
-	trackingOptIn.pushToUserConsentQueue(function () {
-		doc.addEventListener('bab.blocking', function () {
-			if (adContext.get('opts.wadHMD')) {
-				hmdRecLoader.setOnReady(function () {
-					setupPlayer();
-				});
-			} else {
+	function prePlayerSetup(blocking) {
+		if (blocking && adContext.get('opts.wadHMD')) {
+			hmdRecLoader.setOnReady(function () {
 				setupPlayer();
-			}
-		});
+			});
 
-		doc.addEventListener('bab.not_blocking', function () {
-			if (bidders && bidders.isEnabled()) {
-				bidders.runOnBiddingReady(function () {
-					bidParams = bidders.updateSlotTargeting(featuredVideoSlotName);
-					setupPlayer();
-				});
-			} else {
+			return;
+		}
+
+		if (!blocking && bidders && bidders.isEnabled()) {
+			bidders.runOnBiddingReady(function () {
+				bidParams = bidders.updateSlotTargeting(featuredVideoSlotName);
 				setupPlayer();
-			}
-		});
+			});
 
-		if (!adContext.get('opts.showAds')) {
+			return;
+		}
+
+		setupPlayer();
+	}
+
+	trackingOptIn.pushToUserConsentQueue(function () {
+		if (!adContext.get('opts.showAds') || !adContext.get('opts.babDetectionDesktop')) {
```

Comment:
It seems that it skips bidders when bab is disabled

Reply:
You right, fixed.